addons/account_peppol/models/res_company.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    account_peppol_attachment_ids = fields.Many2many(
+        comodel_name='ir.attachment',
+        string='Peppol Documents',
+    )
```

Comment:
I tried, but I couldn't make it stored in `res.config.settings`. It had to be related to a field somewhere else. But since these are deleted immediately after applying, I thought it should be ok?

Reply:
Could you elaborate what kind of error you were getting?

Reply:
No error, the attachments were just not saved, probably because it's a transient model?

Reply:
The transient is still in the database and only garbage collected later. There is nothing preventing it to work in theory

Reply:
Ok, it gets saved. But then the wizard re-opens with a new it and new values; and the action is not called because they expect you to reclick on it after saving.